DOCS/license-inclusion-principles.md

```diff
@@ -1,24 +1,32 @@
 
 ## Candidate License Analysis
 
-Determining whether a candidate license should be included on the SPDX License List requires the SPDX Legal Team to engage in a case-by-case evaluation of each candidate license based on a number of factors. No one factor is dispositive and factors may weight differently. The ultimate decision will be based on the totality of the factors and SPDX's goals and objectives. Factors include:
-* Does the license substantially comply with one of the open source definitions:
-  * Open Source Definition (OSD) from the Open Source Initiative (OSI) - https://opensource.org/osd
-  * Free Software Definition from the Free Software Foundation (FSF) - https://www.gnu.org/philosophy/free-sw.en.html
-  * Open Source Hardware Definition from the Open Source Hardware Association (OSHWA) - https://www.oshwa.org/definition/
-  * Open Definition from the Open Knowledge Foundation (for open data) - http://opendefinition.org/od/2.1/en/
-  * Note: licenses that these organizations consider to meet their definition are presumptively appropriate for inclusion on the SPDX License List. Specifically, SPDX aims to ensure all OSI-approved licenses are included on the SPDX License List and have a short identifier, which is also used on the OSI website.
-* Consider existing use of the license in wild (e.g., wide use of license or use of license in prominent projects)
-* If it is a newly drafted license, it is drafted such that it can be used by anyone (e.g., it is not company-specific or a "vanity" license)
-* Relevant input from the license steward (e.g., regarding name or if the license steward does not want the license added)
-* Relevant open source community factors
-
-Comments on license inclusion will be noted in the Github issue for the license (or reference to meeting minutes, as necessary). Some license requests may be decided solely via comments in the Github issue and some may involve discussion on the bi-weekly legal call. Thus it is recommended that requestors join the mailing list and calls to fully participate.
-
-
-## Backstory
+**Definitive factors:**
+* The submitted license must not match another license already on the SPDX License List as per the SPDX matching guidelines.
+* All OSI-approved licenses will be included on the SPDX License List.
+  * This is consistent with early collaboration between SPDX and OSI (2011) to ensure that all OSI-approved licenses were represented on the SPDX License List. Also att that time, OSI began using the SPDX license identifiers in their URLs and on the license pages.
+* Binary licenses or licenses that do not provide for the avilabilty of the source code will not be included on the SPDX License List.
+  
```

Comment:
do we want to draw a line in the sand on binary licenses? if we are opening up to "source available" then making this restriction seems reasonable. it has effectively been the reality we've operated under all along

Reply:
Please no.  What about data licenses?    I can see cases where folks will want to capture information in SPDX documents - especially training data sets for machine learning, etc.   We want to make it easy for folk to share this sort of info,  not raise barriers.

Reply:
I think the intention is to say "binary-only software licenses won't be added to the list."  E.g. binary-use-only EULAs / software licenses. We already have various open data licenses and there's no intent to close off more of those, but rather to say that the focus for the SPDX License List is licenses that enable sharing of source code.

Reply:
yes, this was about binary software and is consistent with what we've always done.  we could add "software" in there which I think would help clarify any confusion?

Reply:
done - added "software" to modify "binary"